src/cc65/coptmisc.c

```diff
@@ -514,9 +514,10 @@ unsigned OptAXLoad (CodeSeg* S)
         /* Get the next entry */
         const CodeEntry* E = CS_GetEntry (S, I);
 
-        /* Check for incax followed by jsr/jmp ldaxi */
+        /* Check for incax[1-8] followed by jsr/jmp ldaxi */
         if (E->OPC == OP65_JSR                            &&
             strncmp (E->Arg, "incax", 5) == 0             &&
+            strcmp (E->Arg, "incaxy") != 0                &&
```

Comment:
```suggestion
            strncmp (E->Arg, "incax", 5) == 0             &&
            E->Arg[5] >= '1' && E->Arg[5] <= '8'          &&
            E->Arg[6]  == '\0'                            &&
```
It is better to explicitly check for incax[1-8]. Otherwise the code will fail if someone adds an `incaxa` or similar.